website/blog/2024-10-07-release-1.13.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Podman Desktop 1.13 Release
+description: Podman Desktop 1.13 has been released!
+slug: podman-desktop-release-1.13
+authors: SoniaSandler
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.13/banner.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.13 Release! 🎉
+
+![podman-desktop-hero-1.13](img/podman-desktop-release-1.13/banner.png)
+
+Podman Desktop 1.13 is now available. [Click here to download it](/downloads)!
+
+This release includes:
+
+- **HyperV support**: You can now create and manage HyperV Podman machines directly from Podman Desktop.
+- **Search image feature**: Search for an image from Podman Desktop.
+- **Updated empty state pages**: Start your journey in the containers, images, pods, and Kubernetes pages with a click of a button
+- **New Kubernetes navigation**: All Kuberenetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
```

Comment:
typo `Kuberenetes` -> `Kubernetes`

Reply:
@Firewall you can probably use the `suggestion` option when there is a small change :)

```suggestion
- **New Kubernetes navigation**: All Kubernetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
```